index.html

```diff
@@ -27,6 +29,8 @@
         <h2 class="title-bar-item">JSON Path Finder</h2>
       </div>
       <div class="title-bar-item">
+        <a href="#" @click="toggleDarkMode" v-bind:alt='(isDarkMode ? "Light Mode" : "Dark Mode")'><img width="30" height="30" src="./images/contrast-icon.png"></a>
```

Comment:
this will need to be a `button` as links are for navigation. you can add an id to it and add a corresponding selector to the css file to style it to undo the default button styles I put in place. since `alt` won't be available to us anymore on the button, we could use `alt` on the image I guess, plus `title` for a tooltip on it.
```css
#dark-mode-button {
  height: auto;
  width: auto;
  background-color: transparent;
}
```
when modifying `css/style.20210306.css`, you'll need to rename it, replacing the timestamp with the current date in YYYYMMDD format to bust browser+cdn caches.